ote_cli/ote_cli/datasets/anomaly/dataset.py

```diff
@@ -0,0 +1,151 @@
+"""DataLoaders for Anomaly Tasks."""
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import pandas as pd
+from ote_sdk.entities.annotation import (
+    Annotation,
+    AnnotationSceneEntity,
+    AnnotationSceneKind,
+)
+from ote_sdk.entities.dataset_item import DatasetItemEntity
+from ote_sdk.entities.datasets import DatasetEntity
+from ote_sdk.entities.image import Image
+from ote_sdk.entities.label import Domain, LabelEntity
+from ote_sdk.entities.scored_label import ScoredLabel
+from ote_sdk.entities.shapes.rectangle import Rectangle
+from ote_sdk.entities.subset import Subset
+
+
+class AnomalyClassificationDataset(DatasetEntity):
+    """Dataloader for Anomaly Classification Task.
+
+    Example:
+        >>> train_subset = {"ann_file": "data/anomaly/train.json",
+                             "data_root": "data/anomaly/shapes"}
+        >>> val_subset = {"ann_file": "data/anomaly/val.json",
+                            "data_root": "data/anomaly/shapes"}
+        >>> training_dataset = AnomalyClassificationDataset(
+                                train_subset=train_subset,
+                                val_subset=val_subset
+                                )
+        >>> test_subset = {"ann_file": "data/anomaly/test.json",
+                            "data_root": "data/anomaly/shapes"}
+        >>> testing_dataset = AnomalyClassificationDataset(
+                                    test_subset=test_subset
+                                    )
```

Comment:
sorry for being a bit pedantic here, but would love to see if these are formatted via black:)